include/seqan3/alignment/band/static.hpp

```diff
@@ -108,27 +119,41 @@ struct band_static
     band_static & operator=(band_static &&)      = default;
     ~band_static()                               = default;
 
-    //!\brief Construction from seqan3::lower_bound and seqan3::upper_bound.
-    template <typename inner_value_t>
-    constexpr band_static(lower_bound<inner_value_t> const lower, upper_bound<inner_value_t> const upper) noexcept
-        : lower_bound{std::move(lower.get())}, upper_bound{std::move(upper.get())}
+    /*!\brief Construction from seqan3::lower_bound and seqan3::upper_bound.
+     * \tparam input_value_t The input type of the lower and upper diagonal indices.
+     * \param lower The lower diagonal that defines the band; must be integral.
+     * \param upper The upper diagonal that defines the band; must be integral.
+     *
+     * \attention The lower diagonal index is a positive number that defines the lower boundary of the band
+     * in south direction from the origin. Negative values are converted to their absolute value.
+     * Analogously, the upper diagonal index is a positive number that defines the upper boundary of the band
+     * in east direction from the origin.
+     */
+    template <std::Integral input_value_t>
+    constexpr band_static(lower_bound<input_value_t> const lower, upper_bound<input_value_t> const upper) noexcept
+        : lower_bound{lower.get() >= 0 ? lower.get() : -lower.get()},
+          upper_bound{upper.get() >= 0 ? upper.get() : -upper.get()}
```

Comment:
Is there a reason why we make this positive? I discussed this with @rrahn  a couple of weeks ago and he said that he wanted to keep the negative value. Did he change his opinion? We should discuss this with again.

If you don't allow negative values, you are not able to generate matrices like this:
```c++
        //     e,  A,  A,  C,  C,  G,  G,  T,  T,  A,  A,  C,  C,  G,  G,  T,  T
        /*e*/  0,  1,  2,  3,  4,  5,  6,  7,  8,INF,INF,INF,INF,INF,INF,INF,INF,
        /*A*/INF,INF,INF,INF,  4,  5,  6,  7,  8,  8,INF,INF,INF,INF,INF,INF,INF,
        /*C*/INF,INF,INF,INF,INF,  5,  6,  7,  8,  9,  9,INF,INF,INF,INF,INF,INF,
        /*G*/INF,INF,INF,INF,INF,INF,  5,  6,  7,  8,  9, 10,INF,INF,INF,INF,INF,
        /*T*/INF,INF,INF,INF,INF,INF,INF,  5,  6,  7,  8,  9, 10,INF,INF,INF,INF,
        /*A*/INF,INF,INF,INF,INF,INF,INF,INF,  6,  6,  7,  8,  9, 10,INF,INF,INF,
        /*C*/INF,INF,INF,INF,INF,INF,INF,INF,INF,  7,  7,  7,  8,  9, 10,INF,INF,
        /*G*/INF,INF,INF,INF,INF,INF,INF,INF,INF,INF,  8,  8,  8,  8,  9, 10,INF,
        /*T*/INF,INF,INF,INF,INF,INF,INF,INF,INF,INF,INF,  9,  9,  9,  9,  9, 10,
        /*A*/INF,INF,INF,INF,INF,INF,INF,INF,INF,INF,INF,INF, 10, 10, 10, 10, 10
```